packages/issuer/lib/VcIssuer.ts

```diff
@@ -24,13 +26,16 @@ import {
   TYP_ERROR,
 } from '@sphereon/openid4vci-common'
 import { ICredential, W3CVerifiableCredential } from '@sphereon/ssi-types'
+import { v4 } from 'uuid'
 
 export class VcIssuer {
   _issuerMetadata: IssuerMetadata
   _userPinRequired?: boolean
   _issuerCallback?: CredentialIssuerCallback
   _verifyCallback?: JWTVerifyCallback
   private readonly _stateManager: ICredentialOfferStateManager
+  private readonly _cNonce: string[] = []
```

Comment:
Plural, also we need to create an interface for a nonce manager or something. Since this needs to be extended in the future with a persistent store, similar like for the credential offer state manager.

It is not the concern of the low level lib to provide all kinds of implementations, but we should be able to start doing proper persistence for instance in our SDK